README.md

```diff
@@ -1,40 +1,133 @@
-# Getting started with network socket statistics on Mbed OS
+![](./resources/official_armmbed_example_badge.png)
+# Socket Statistics Mbed OS Example
 
-This example demonstrates how you can collect statistics from network sockets. You can enable statistics with the `nsapi.socket-stats-enabled` configuration option in `mbed_app.json`:
+This guide reviews the steps required to get socket statistics on an Mbed OS enabled platform.
 
-```
-{
-    "target_overrides": {
-        "*": {
-            "nsapi.socket-stats-enabled": true
-        }
-    }
-}
-```
+You can refer [mbed-os-example-sockets](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md) example for more information about network socket setup.
+
+You can build the project with all supported [Mbed OS build tools](https://os.mbed.com/docs/mbed-os/latest/tools/index.html). However, this example project specifically refers to the command-line interface tool [Arm Mbed CLI](https://github.com/ARMmbed/mbed-cli#installing-mbed-cli).
+
+1. [Install Mbed CLI](https://os.mbed.com/docs/mbed-os/latest/quick-start/offline-with-mbed-cli.html).
+
+1. Clone this repository on your system, and change the current directory to where the project was cloned:
+
+    ```bash
+    $ git clone git@github.com:armmbed/mbed-os-example-socket-stats && cd mbed-os-example-socket-stats
+    ```
+
+    Alternatively, you can download the example project with Arm Mbed CLI using the `import` subcommand:
+
+    ```bash
+    $ mbed import mbed-os-example-socket-stats && cd mbed-os-example-socket-stats
+    ```
+
+## Application functionality
+
+The `main()` function gets the default instance of a network interface and set up the socket connection. It also starts a thread to print socket statistics such as  `reference_id`, `state`, `proto`, `sent_bytes`, `recv_bytes`, `last_change_tick` on the serial interface.
+
```

Comment:
The current example is configured to use the Ethernet interface on supported devices. To use the example with a different interface, please follow the [socket example readme](https://github.com/ARMmbed/mbed-os-example-sockets/blob/master/README.md).